kotlin/toolchains.bzl

```diff
@@ -111,6 +119,11 @@ def define_kt_toolchain(name, language_version=None, api_version=None, jvm_targe
         api_version = api_version,
         jvm_target = jvm_target,
         coroutines = coroutines,
+        jvm_strict_deps = select({
```

Comment:
@brendandouglas FYI this is how I am currently going to relay the strict dep flag to the builder. Is this the right place for it ?  My initial thoughts were to go through a common hidden attribute on the rules but https://github.com/bazelbuild/bazel/issues/287 prevents this. I can make it hidden here ? 

p.s., This branch is quite chaotic atm so don't bother reading any of the other code.